utils/logger.cpp

```diff
@@ -20,4 +20,6 @@
 // declare a global logger
 namespace cosa {
 Log logger;
+
+void set_global_logger_verbosity(unsigned int v) { logger.set_verbosity(v); }
```

Comment:
clang format

Reply:
Ran it again, looks like that's what clang-format does actually.

Reply:
ok :)